src/containers/MultidisciplinarySubject/MultidisciplinarySubjectArticle.jsx

```diff
@@ -99,4 +100,18 @@ MultidisciplinarySubjectArticle.propTypes = {
   locale: PropTypes.string,
 };
 
-export default MultidisciplinarySubjectArticle;
+MultidisciplinarySubjectArticle.getDocumentTitle = ({ t, data }) => {
+  return `${data?.topic?.name || ''}${t('htmlTitles.titleTemplate')}`;
+};
+
+MultidisciplinarySubjectArticle.willTrackPageView = (
+  trackPageView,
+  currentProps,
+) => {
+  const { data, loading } = currentProps;
+  if (!loading && data?.topic?.article) {
+    trackPageView(currentProps);
+  }
```

Comment:
Dette vil aldri slå til fordi data hentes inni komponenten og sendes ikkje inn som props utenfra. 

For å få til dette må du lage en MultidisciplinarySubjectArticleWrapper som gjør graphql-spørringa og som igjen har en MultidisciplinarySubjectArticle som tar imot data som props. Då vil denne koden kunne trigges.

Men eg har andre endringer i denne fila i https://github.com/NDLANO/ndla-frontend/pull/624 som det hadde vore en fordel å merge inn først.